libs/contrib/Data/Fun/Extra.idr

```diff
@@ -62,19 +62,19 @@ applyPartially fgs = uncurry {ts} {cod = Fun ss cod} (homoFunMult_ext {rs=ts} {s
 
 ||| Apply an n-ary dependent function to its tuple of inputs (given by an HVect)
 public export
-uncurryAll : {ts : Vect n Type} -> {cod : Fun ts Type} 
+uncurryAll : {n : Nat} -> {0 ts : Vect n Type} -> {0 cod : Fun ts Type} 
```

Comment:
Same here wrt. to `HVect` giving you a handle on `n`.

Reply:
Done.